### ISSUE TITLE
setValue:forUndefinedKey

### ISSUE THREAD

**Issue by aliencode** [MengTo/Spring#33]
```
2015-03-03 19:55:52.507 demos[24988:4604020] Failed to set (animation) user defined inspected property on (UIView): [<UIView 0x7a8860b0> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key animation.

```

**Comment by jamztang** [comment]
You should subclass your `UIView` to `SpringView`

**Comment by Scrivs** [comment]
I get a similar error when running on Xcode 6.1.1 and Xcode 6.3. This is with brand new projects from scratch include trying the Spring Xcode project.

2015-03-09 19:37:01.672 betaDebug[71870:1269910] Failed to set (damping) user defined inspected property on (UIButton): [<UIButton 0x7fd7ead57480> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key damping.

**Comment by jamztang** [comment]
Do you mean you couldn't run the sample project?

**Comment by Scrivs** [comment]
Correct. 

After more investigating and trying the project in different directories I'll either get Build errors, compile errors, or warnings. Depending on the where I install the project it just seems to be a crapshoot as to what errors I'll get.

I'm thinking this is bugginess on Xcode/Swift compiler side. I'm still investigating to see if there is a true fix and I'll be sure to report back.

**Comment by Scrivs** [comment]
For documentation purposes here is a compiler error I'll get just by running the sample Spring project:

`Value of optional type 'UIFont?' no unwrapped; did you mean to use '!' or '?'?`

**Comment by jamztang** [comment]
What Xcode version you're using?

**Comment by Scrivs** [comment]
Testing both 6.1.1 and 6.3.

**Comment by jamztang** [comment]
6.1.1 was working fine for me.
6.3 will not work since it has new Swift 1.2 syntax, we'll wait until it's out of beta.
I am upgrading to 6.2 and make sure everything is fine.

**Comment by Scrivs** [comment]
Got ya. I'll go back to 6.1.1 and see what I come up with.

**Comment by jamztang** [comment]
sure. Ok 6.2 should also works fine without changing any lines of code.

**Comment by vlafiser** [comment]
Yep, I have the same issue. Thank you for informations!

**Comment by Scrivs** [comment]
Alright so here is my progress report before I go to bed. I think once a project gets "corrupted" things just get screwed up. I'm not going to pretend to know how Xcode's compiler works or if it remembers compile errors from other projects, but this is what finally got this stuff working for me.

* Downloaded a brand new version of Spring Framework. Delete all previous folders.
* Downloaded Xcode 6.2 (I don't think this really mattered)
* Created a new project in a completely new folder.
* Imported the Spring framework by dragging the new downloaded folder into my project.
* Everything worked.

So as of now my guess is that while testing the Spring Framework out before, Xcode did one of its funky bugs so whenever I imported the Spring Framework folder into my other projects it brought that mess along.

I could be completely wrong, as again, I'm not a compiler expert so I don't know what data travels with the folder.

But I'm glad I got it working again.
